api/graphite.go

```diff
@@ -0,0 +1,350 @@
+package api
+
+import (
+	"errors"
+	"net/http"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/raintank/dur"
+	"github.com/raintank/metrictank/api/middleware"
+	"github.com/raintank/metrictank/api/models"
+	"github.com/raintank/metrictank/api/rbody"
+	"github.com/raintank/metrictank/cluster"
+	"github.com/raintank/metrictank/consolidation"
+	"github.com/raintank/metrictank/idx"
+	"github.com/raintank/worldping-api/pkg/log"
+	"gopkg.in/raintank/schema.v1"
+)
+
+var MissingOrgHeaderErr = errors.New("orgId not set in headers")
+var MissingQueryErr = errors.New("missing query param")
+var InvalidFormatErr = errors.New("invalid format specified")
+var MaxPointsPerReqErr = errors.New("request would execed maxPointsPerReq setting")
+var MaxDaysPerReqErr = errors.New("request would execed maxDaysPerReq setting")
```

Comment:
Firstly, i will change the response code for these errors to 403 (Forbidden) as the request doesnt contain bad data, it is just forbidden by the configuration.

I dont agree that the old message was better.  "too many targets/maxDataPoints requested"  isnt that helpful.   does it mean targets divided by maxDataPoint? Shouldnt it be targets*maxDatapoints?

What about: "request exceeds max-points-per-req limit. Reduce the number of series and or reduce the maxDataPoints requested or ask your admin to increase the limit."